src/components/CheckBox/CheckBox.tsx

```diff
@@ -35,7 +43,27 @@ export const CheckBox: FC<CheckboxProps> = React.forwardRef(
 
         const checkboxWrapperClassNames: string = mergeClasses([
             styles.selector,
+            {
+                [styles.selectorSmall]:
+                    size === SelectorSize.Flex && largeScreenActive,
+            },
+            {
+                [styles.selectorMedium]:
+                    size === SelectorSize.Flex && mediumScreenActive,
+            },
```

Comment:
someone had suggested combining these in a different PR. Not sure if that works. Personally think we should revisit mergeClasses. classNames provides more flexibility:
https://www.npmjs.com/package/classnames

Reply:
Ya, this was an idea for the Atom component.